packages/web-components/src/components/cbp-toast/cbp-toast.stories.tsx

```diff
@@ -22,17 +22,18 @@ export default {
     },
   };
   
-  const Template = ({ open, icon, content, buttons, duration, color, context, sx }) => {
+  const Template = ({ open, icon, title, content, buttons, duration, color, context, sx }) => {
     return ` 
           <cbp-toast
-            ${open ? `open=${open}` : ''}
-            color=${color}
-            duration=${duration}
+            ${open ? `open` : ''}
+            ${color ? `color=${color}` : ''}
+            ${duration ? `duration=${duration}` : ''}
             ${icon ? `icon=${icon}` : ''}
             ${context && context != 'light-inverts' ? `context=${context}` : ''}
             ${sx ? `sx=${JSON.stringify(sx)}` : ''}
-          >
-            <div slot="cbp-toast-title"> Test Toast Title</div>
+          >  
+            <div slot="cbp-toast-icon">${icon}</div>
```

Comment:
I would just pass the icon name here, not the whole icon component. Just code the icon component into the story. And remember that you cannot self-close web components/custom elements (only in JSX). You need a full closing tag.

Reply:
updated as requested, this seems to be where the rogue > was coming from as well